src/Components/Dropdown/Dropdown.js

```diff
@@ -11,50 +11,64 @@ import {
   ArrowsContainer
 } from './Dropdown.styles';
 
-export const Dropdown = ({ items = [] }) => {
-  const [open, setOpen] = useState(false);
-  const [selection, setSelection] = useState('USD');
-  const toggle = () => setOpen(!open);
+export class Dropdown extends React.Component {
+  state = {
+    isOpen: false,
+    selection: 'USD'
+  };
+  toggle = () =>
+    this.setState((prevState) => ({
+      ...this.prevState,
+      isOpen: !prevState.isOpen
+    }));
 
-  useEffect(() => {
+  componentDidMount = () => {
     const CurrentSelection = localStorage.getItem('selection');
```

Comment:
add a default in case you don't have selection in local storage and you get null back ;)